src/ConsoleContactAppManager/Validators.cs

```diff
@@ -0,0 +1,122 @@
+﻿// <copyright file="Validators.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace ContactManagerApp
+{
+    using System.Text.RegularExpressions;
+
+    /// <summary>
+    /// Class validators have the methods to check if the fields enetered by user are valid.
+    /// </summary>
+    internal static class Validators
+    {
+        /// <summary>
+        /// The CheckIfEmpty method checks if the contact list is empty.
+        /// </summary>
+        /// <param name="contactList"> The contact list is passed as input </param>
+        /// <returns>Returns 0 if there is no contact in the list and 1 if the list has contacts</returns>
+        public static int CheckIfEmpty(List<Contact> contactList)
```

Comment:
You don't need a method for this as this can be checked with a single if statement.

Reply:
This method is removed.